e2e/Uno/HelloUnoWorld.Shared/App.xaml.cs

```diff
@@ -0,0 +1,129 @@
+﻿using HelloUnoWorld.Views;
+using HelloUnoWorld.Dialogs;
+using Microsoft.Extensions.Logging;
+using Prism;
+using Prism.DryIoc;
+using Prism.Ioc;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+namespace HelloUnoWorld
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : PrismApplication
+    {
+        /// <summary>
+        /// Initializes the singleton application object.  This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            ConfigureFilters(global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory);
+
+            this.InitializeComponent();
```

Comment:
this is typically something we would want to encourage people to do in the OnInitialized from PrismApplication

Reply:
In the context of Uno, doing this in `OnLaunched` is too late to get meaningful logging for the startup path, hence the placement in the constructor.

Reply:
I agree OnLaunched would be too late... but OnInitialized should be called as part of the PrismApplicationBase ctor initialization. It's more of an alignment that rather than trying to stuff everything in the ctor of your app you should place that initialization logic in PrismApplication's OnInitialized.

Reply:
Moving `OnInitialized` from `OnLaunched` to the ctor is a significant change from the WPF behavior (which does it on `OnStartup`), but I don't see any problem doing so.

Reply:
In case you've read the comment too early, I've edited my comment above, and I'd suggest keeping the WPF behavior.

Reply:
you are right... I'll mark this issue as resolved.